rd_stats.h

```diff
@@ -187,6 +187,10 @@ struct stats_paging {
 	unsigned long pgscan_kswapd	__attribute__ ((aligned (8)));
 	unsigned long pgscan_direct	__attribute__ ((aligned (8)));
 	unsigned long pgsteal		__attribute__ ((aligned (8)));
+	unsigned long pgpromote_success __attribute__ ((aligned (8)));
```

Comment:
Please rename this field to `pgpromote`. We don't have several fields for the number of pages promoted.